static/app/views/performance/newTraceDetails/virtualizedViewManager.tsx

```diff
@@ -388,6 +381,27 @@ export class VirtualizedViewManager {
     this.draw();
   }
 
+  registerContainerRef(container: HTMLElement | null) {
+    if (container) {
+      this.initialize(container);
+    } else {
+      this.teardown();
+    }
+  }
+
+  registerGhostRowRef(column: string, ref: HTMLElement | null) {
```

Comment:
Since we are not storing row references anywhere, the event listeners should naturally be removed once the row unmounts.